From 5af1d2674030fcfb834a702c2ebf475a4944b3a5 Mon Sep 17 00:00:00 2001
From: Xu Mao <b34688@freescale.com>
Date: Fri, 14 Dec 2012 15:56:16 +0800
Subject: [PATCH 125/304] ENGR00133115 add https_pipe_v3 , rtps_pipe and
 udps_pipe

add https_pipe_v3 , rtps_pipe and udps_pipe

Signed-off-by: Xu Mao <b34688@freescale.com>
---
 imx5x/imx5x.mk | 3 +++
 imx6/imx6.mk   | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/imx5x/imx5x.mk b/imx5x/imx5x.mk
index 105d4cf..7716ffa 100644
--- a/imx5x/imx5x.mk
+++ b/imx5x/imx5x.mk
@@ -180,6 +180,9 @@ omx_libs := \
 	lib_omx_android_audio_render_arm11_elinux	\
 	lib_omx_audio_fake_render_arm11_elinux		\
 	lib_omx_ipulib_render_arm11_elinux		\
+	lib_omx_https_pipe_v3_arm11_elinux		\
+	lib_omx_udps_pipe_arm11_elinux			\
+	lib_omx_rtps_pipe_arm11_elinux			\
 	lib_avi_parser_arm11_elinux.3.0			\
 	lib_divx_drm_arm11_elinux			\
 	lib_mp4_parser_arm11_elinux.3.0			\
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index eacdcfd..9846f99 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -217,6 +217,9 @@ omx_libs := \
 	lib_omx_vpu_enc_v2_arm11_elinux \
 	lib_ffmpeg_arm11_elinux	\
 	lib_omx_https_pipe_v2_arm11_elinux \
+	lib_omx_https_pipe_v3_arm11_elinux \
+	lib_omx_udps_pipe_arm11_elinux \
+	lib_omx_rtps_pipe_arm11_elinux \
 	lib_omx_streaming_parser_arm11_elinux \
 	lib_omx_surface_render_arm11_elinux \
 	libfsl_jpeg_enc_arm11_elinux \
-- 
1.8.0

